### DISCUSSION TITLE
What does 'show IDE completions side-by-side' mean?

### DISCUSSION THREAD

**Discussion by c2ray** [community/community#11474] [Copilot Conversations] [answered]
What does it mean?
I checked this option, but found nothing different
![image](https://user-images.githubusercontent.com/55729509/153811024-fbe18ecc-073a-4fe6-89bf-b8d4b0d824f1.png)

**Comment by jansorg** [comment, marked as answer]
With this option enabled, you'll be able to see both the IDE's completion popup and GitHub Copilot's completions at the same time in the editor. Without it, the popup is not shown when Copilot has a suggestion.

**Reply by alanhe421** [reply]
So is the practice on or off?

Thank you.
